server/emails/helpers.js

```diff
@@ -0,0 +1,10 @@
+// checks if the date of the appointment is tomorrow
+
+const isAppointmentTomorrow = date => {
+  const currentDate = new Date();
+  const tomorrowDate = new Date(currentDate.setDate(currentDate.getDate() + 1));
+  const apptDate = new Date(date);
+  return apptDate.getDate() === tomorrowDate.getDate();
```

Comment:
@lucymk I think you should change this to comparing the date string like you had before